test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsEndToEndTestsBase.cs

```diff
@@ -294,6 +295,11 @@ public async Task Validate_HostLogs()
             ValidateTrace(traces[idx++], "Job host started", LogCategories.Startup);
             ValidateTrace(traces[idx++], "Loading functions metadata", LogCategories.Startup);
             ValidateTrace(traces[idx++], "Reading functions metadata", LogCategories.Startup);
+
+            // The dotnet-isolated worker will not be loaded for this test as it is setup to run only in placeholder mode.
+            ValidateTrace(traces[idx++], "Skipping WorkerConfig for stack: dotnet-isolated since it is disabled.", LanguageWorkerConfigLogCategory);
+            // TO DO: Investigate why the message is logged twice. Tracking issue: https://github.com/Azure/azure-functions-host/issues/10268
+            ValidateTrace(traces[idx++], "Skipping WorkerConfig for stack: dotnet-isolated since it is disabled.", LanguageWorkerConfigLogCategory);
```

Comment:
I see the code path which populates the worker config is executed more than once (and hence logging the log message twice). I opened an issue to investigate that: https://github.com/Azure/azure-functions-host/issues/10268

Reply:
made an update to test to ignore the skip message. So this comment is not relevant anymore. I will still investigate the double execution is happening in non-test scenarios (the test is executing it twice one from the test fixture creation and one from the test)